lib/archive_importer.rb

```diff
@@ -25,10 +25,13 @@ def create_user(attr)
       email strip_exif show_community_spotlight_in_stream language disable_mail auto_follow_back
     ]
     data = convert_keys(archive_hash["user"], allowed_keys)
+    # setting getting_started to false as we are importing an existing user profile
+    # so the user need not see the getting_started wizard
```

Comment:
I'm not a native speaker but I feel like "the user doesn't need to see the getting started wizard" sounds better.